src/Android/Renderers/CustomEditorRenderer.cs

```diff
@@ -12,6 +12,15 @@ public class CustomEditorRenderer : EditorRenderer
         public CustomEditorRenderer(Context context)
             : base(context)
         { }
+        
+        // Workaround for issue described here:
+        // https://github.com/xamarin/Xamarin.Forms/issues/8291#issuecomment-617456651
+        protected override void OnAttachedToWindow()
+        {
+            base.OnAttachedToWindow();
+            EditText.Enabled = false;
```

Comment:
For the love of all that's created, please add a code comment as to where this hack is coming from :-)